### ISSUE TITLE
filterVideoAndAudio() Stream URL Returns 403 Error

### ISSUE THREAD

**Issue by meirinberg** [alexeichhorn/YouTubeKit#48]
Hello, I've followed Example 1 explicitly and my video does not load. Interestingly, the audio-only and video-only streams do work successfully. After some debugging, I discovered that the returned URL of the combined video and audio stream has a 403 error from googlevideo.com. Why would a 403 Forbidden error occur for the combined stream but not the video-only or audio-only streams? 

Here's a refresher on the code snippet I've been referencing:
Example 1: 403 Error for returned URL when .filterVideoAndAudio() is used; but .filterVideoOnly(), for example, plays successfully.
```
let stream = try await YouTube(videoID: "QdBZY2fkU-0").streams
                          .filterVideoAndAudio()
                          .filter { $0.isNativelyPlayable }
                          .highestResolutionStream()

let player = AVPlayer(url: stream!.url)
```

I'd be happy to provide more information, thanks in advance.

**Comment by alexeichhorn** [comment]
Did you use the latest version or main branch directly?
I have a fix on main branch that is not published as a new version yet. Will publish it today/tomorrow, if nobody finds bugs.

**Comment by meirinberg** [comment]
FYI: I updated the "Cipher" and "Extraction" swift files locally to match the latest GitHub version -- and this resolved my issue.

**Comment by alexeichhorn** [comment]
Great. I also now released the changes as official version 0.2.3.

**Comment by meirinberg** [comment]
> Great. I also now released the changes as official version 0.2.3.

Thanks so much!
